routes/api/current.router.js

```diff
@@ -0,0 +1,51 @@
+const express = require("express");
+const current = express.Router();
+const User = require("../../model/user.model");
+const tokenMiddleware = require("../../middleware/tokenMiddleware");
+
+current.get("/", tokenMiddleware, async (req, res) => {
+  try {
+    const currentUser = await User.findById(req.user._id);
```

Comment:
Tego current juz masz, bo został on pobrany w tokenMiddleware i przypisany do `req.user` więc nie musisz tego drugi raz robić i sprawdzać. W zasadzie Jeżeli ktoś był w stanie dotrzeć do tego miejsca to możesz spokojnie zwrócić mu informację. :)